chainntnfs/tx_notifier.go

```diff
@@ -1164,6 +1198,38 @@ func (n *TxNotifier) dispatchConfReorg(ntfn *ConfNtfn, heightDisconnected uint32
 	return nil
 }
 
+// dispatchSpendReorg dispatches a reorg notification to the client if a spend
+// notiification was already delivered.
+//
+// NOTE: This must be called with the TxNotifier's lock held.
+func (n *TxNotifier) dispatchSpendReorg(ntfn *SpendNtfn) error {
+	if !ntfn.dispatched {
+		return nil
+	}
+
+	// Attempt to drain the spend notification to ensure sends to the Spend
+	// cahnnel are always non-blocking.
+	select {
+	case <-ntfn.Event.Spend:
+	case <-n.quit:
```

Comment:
nit: selecting on quit not needed here when we have default clause.

Reply:
Fixed.